packages/components/src/date-time/style.scss

```diff
@@ -1,262 +1 @@
-			padding-right: 0;
-		}
-	}
-
-	&.is-description-visible .DayPicker {
-		visibility: hidden;
-	}
-}
-
-.components-datetime__date .CalendarDay .components-datetime__date__day {
-	height: 100%;
-	display: flex;
-	justify-content: center;
-	align-content: center;
-	flex-direction: column;
-	position: relative;
-
-	&.has-events::before {
-		content: " ";
-		width: 4px;
-		height: 4px;
-		border-radius: 2px;
-		position: absolute;
-		left: 50%;
-		margin-left: -2px;
-		bottom: 0;
-		background-color: $white;
-	}
-}
-
-.components-datetime__date .CalendarDay:not(.CalendarDay__selected) .components-datetime__date__day.has-events::before {
-	background: var(--wp-admin-theme-color);
-}
-
-.components-datetime__time {
-	padding-bottom: $grid-unit-20;
-
-	fieldset {
-		position: relative;
-		margin-bottom: 0.5em;
-	}
-
-	fieldset + fieldset {
-		margin-bottom: 0;
-	}
-
-	.components-datetime__time-field-am-pm fieldset {
-		margin-top: 0;
-	}
-
-	.components-datetime__time-wrapper {
-		display: flex;
-
-		.components-datetime__time-separator {
-			display: inline-block;
-			padding: 0 3px 0 0;
-		}
-
-		.components-datetime__time-field {
-
-			&-time {
-				/*rtl:ignore*/
-				direction: ltr;
-			}
-
-			select {
-				margin-right: $grid-unit-05;
-
-				&:focus {
-					position: relative;
-					z-index: 1;
-				}
-			}
-
-			input[type="number"] {
-				padding: 2px;
-				margin-right: $grid-unit-05;
-				text-align: center;
-				-moz-appearance: textfield;
-
-				&:focus {
-					position: relative;
-					z-index: 1;
-				}
-
-				&::-webkit-inner-spin-button {
-					-webkit-appearance: none;
-					margin: 0;
-				}
-			}
-		}
-	}
-
-	// Makes the month appear before the day if time format uses AM/PM
-	// We are assuming MM-DD-YYY correlates with AM/PM
-	&.is-12-hour {
-		.components-datetime__time-field-day input {
-			margin: 0 -$grid-unit-05 0 0 !important;
-			border-radius: $radius-block-ui 0 0 $radius-block-ui !important;
-		}
-		.components-datetime__time-field-year input {
-			border-radius: 0 $radius-block-ui $radius-block-ui 0 !important;
-		}
-	}
-}
-
-.components-datetime__timezone {
-	line-height: 30px;
-	margin-left: $grid-unit-05;
-	text-decoration: underline dotted;
-}
-
-.components-datetime__time-legend {
-	font-weight: 600;
-	margin-top: 0.5em;
-
-	&.invisible {
-		position: absolute;
-		top: -999em;
-		left: -999em;
-	}
-}
-
-.components-datetime__time-field-integer-field {
-	font-family: inherit;
-}
-
-.components-datetime__time-field-hours-input,
-.components-datetime__time-field-minutes-input,
-.components-datetime__time-field-day-input {
-	width: 35px;
-}
-
-.components-datetime__time-field-year-input {
-	width: 55px;
-}
-
-.components-datetime__time-field-month-select {
-	max-width: 145px;
-}
-
-// Hack to center the datepicker component within the popover.
-// It sets its own styles so centering is tricky.
-.components-popover .components-datetime__date {
-	padding-left: $grid-unit-05;
-}
+@import "./date/style.scss";
```

Comment:
> * `date-time/date/datepicker.scss` is basically a copy-paste of the original styles in `react-dates`

Correct.

> * plus a few tweaks

I don't actually think there are any tweaks. I'm not really sure why we vendor the stylesheet instead of using the one that's assumedly in `node_modules`.

> * `date-time/styles.scss` just imports the styles mentioned above

Correct.

> * `date-time/styles.ts` contains all remaining styles for the components written with Emotion

The `styles.ts` files are all within the component subfolders, so it's `date-time/date/styles.ts`.

---

> * rename `date-time/date/datepicker.scss` to `date-time/date/lib.scss`

Sounds good. I'll first quickly check to see we can just use what's in `node_modules`.

> * move the code in `date-time/date/style.scss` to `date-time/styles.scss` (which would import `./date/lib.scss`)

No strong preference here. I suppose it's one less "go to definition" command you have to do when following the code, so yep sure!

Reply:
Your plan sounds good to me!

> I don't actually think there are any tweaks. I'm not really sure why we vendor the stylesheet instead of using the one that's assumedly in node_modules.

The reason why I mentioned that is because of this comment at the start of the file: https://github.com/WordPress/gutenberg/blob/2aadd5fcea1c15dd6768886d8af635de5780b362/packages/components/src/date-time/date/datepicker.scss#L2-L3

Reply:
OK looked more into this and there's some history on why we have a vendored copy of `datepicker.scss`: https://github.com/WordPress/gutenberg/pull/26534 Basically it uses `darken()` for some reason and so needs to have the `.scss` extension. Vendoring it is the easiest way to do so.

I also am thinking it's still best to keep both `date-time/date/style.scss` and `date-time/style.scss` as it makes it more explicit that only `DatePicker` uses `react-dates`.

Note to self: Really ought to investigate using something other than `react-dates` (maybe `react-calendar`).

Reply:
Thank you for the context!

> Really ought to investigate using something other than `react-dates`

That would be great!

I personally would also love to find an alternative to `moment`, mostly to save on bundle size